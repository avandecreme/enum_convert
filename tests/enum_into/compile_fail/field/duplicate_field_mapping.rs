use enum_convert::EnumInto;

#[derive(EnumInto)]
#[enum_into(Target)]
enum Source {
    Struct {
        // Should be #[enum_into(Target::Struct.a)]
        #[enum_into(Target::Struct.a, Target::Struct.b)]
        x: i32,
    },
}

enum Target {
    Struct { a: i32 },
}

fn main() {}
