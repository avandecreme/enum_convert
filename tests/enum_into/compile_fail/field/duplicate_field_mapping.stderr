error: Multiple mapping found for target enum `Target`
 --> tests/enum_into/compile_fail/field/duplicate_field_mapping.rs:8:9
  |
8 |         #[enum_into(Target::Struct.a, Target::Struct.b)]
  |         ^
